### DISCUSSION TITLE
How to set border-width?

### DISCUSSION THREAD

**Discussion by JAN-CHOU** [six7/figma-tokens#588] [Q&A] [answered]
I want to know how to set border-width using plugin, the method I set up now needs to be done through figma's native settings, I need to use figma for the stroke to work so I can change the border-width using the plugin, but I have no way to set the stroke color directly, I can only change the color of the stroke through figma😿

**Comment by six7** [comment, marked as answer]
You can right click a color token to apply that one as a stroke color.

And for borderWidth there's a dedicated borderWidth token, which you'd just apply. Ideally you'd use the two in combination.

**Reply by JAN-CHOU** [reply]
Okay, great, thanks!
